src/models/edge.ts

```diff
@@ -46,6 +59,7 @@ export type IEdgeStyle = Partial<{
   widthHover: number;
   widthSelected: number;
   zIndex: number;
+  edgeLineStyle: IEdgeLineStyle;
```

Comment:
No need to have `edge` because you already have a context of `EdgeStyle`. I would suggest naming this `lineStyle`.

Reply:
Renamed it to `lineStyle`, pls check.